examples/messages-from-gchat/add-to-room.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please rename this directory to `/hangouts-chat-messages/` for consistency. GChat is colloquial for Google Talk.

Reply:
Will do.